content_schemas/formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -1551,6 +1552,7 @@
             "eritrea",
             "estonia",
             "ethiopia",
+            "federated-states-of-micronesia",
```

Comment:
No idea why in the corresponding json for this `content_schemas/examples/specialist_document/frontend/international-development-funding.json` not all countries are listed as allowed values. Seems to work fine though. 🥲